packages/at_secondary_server/test/monitor_verb_test.dart

```diff
@@ -360,6 +361,7 @@ void main() {
           ',"deviceName":"$deviceName"'
           ',"namespaces":${jsonEncode(namespaces)}'
           ',"apkamPublicKey":"dummy_apkam_public_key"'
+          ',"encryptedAPKAMSymmetricKey":"dummy_encrypted_apkam_symm_key"'
```

Comment:
@murali-shris Looks like we need to tighten up validation of enrollment requests to ensure that all required data is present. Suggest open another ticket and do another PR for that